src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2040,6 +2040,15 @@ private BoundExpression BindIncrementOperator(CSharpSyntaxNode node, ExpressionS
 
         private BoundExpression BindSuppressNullableWarningExpression(PostfixUnaryExpressionSyntax node, DiagnosticBag diagnostics)
         {
+            if (this.Flags.Includes(BinderFlags.AttributeArgument))
+            {
+                diagnostics.Add(new LazyMissingNonNullTypesContextForSuppressionDiagnosticInfo(NonNullTypesContext), node.OperatorToken.GetLocation());
+            }
+            else if (NonNullTypesContext.NonNullTypes == null)
+            {
+                diagnostics.Add(ErrorCode.WRN_MissingNonNullTypesContext, node.GetLocation());
```

Comment:
The location of the error is different between the two cases.

Reply:
Fixed. Thanks